operator/src/main/java/oracle/kubernetes/operator/helpers/DomainPresenceInfoManager.java

```diff
@@ -24,9 +22,12 @@ public static DomainPresenceInfo getOrCreate(String ns, String domainUID) {
   }
 
   public static DomainPresenceInfo getOrCreate(Domain domain) {
+    DomainPresenceInfo oldInfo = lookup(domain.getSpec().getDomainUID());
+    DomainPresenceInfo oldInfo2 = domains.get(domain.getSpec().getDomainUID());
     DomainPresenceInfo createdInfo = new DomainPresenceInfo(domain);
     DomainPresenceInfo existingInfo =
         domains.putIfAbsent(domain.getSpec().getDomainUID(), createdInfo);
+    if (existingInfo == null && oldInfo != null) System.out.println("This can't be");
```

Comment:
Odd...  can we use a different pattern than System.out?  throw new IllegalStateException()?

Reply:
Um... that was a debug statement. It doesn't belong there at all. Oops.